src/lifecycle.js

```diff
@@ -269,21 +253,22 @@ function addEventListeners (target, component) {
  * @returns {undefined}
  */
 function triggerCreated (target, component) {
-  if (ensureLifecycleFlag(target, component, 'created')) {
+  var targetData = data(target);
+
+  if (targetData.get('created')) {
     return;
   }
 
+  targetData.set('created', true);
+
   // TODO: This doesn't need to happen if using native.
   inherit(target, component.prototype, true);
 
   // We use the unresolved / resolved attributes to flag whether or not the
-  // element has been templated or not. The resolved attribute is also how
-  // you can ensure there isn't a FOUC. If no templating is happening, then
-  // there is no reason to add / remove these attributes.
-  if (component.template && !target.hasAttribute(component.resolvedAttribute)) {
```

Comment:
Why'd you move away from using `resolvedAttribute`? Does this mean that #133 (prerendering things server-side) should be reopened?